ports/antonballclassic/port.json

```
@@ -0,0 +1,31 @@
+{
+  "version": 3,
+  "name": "antonballclassic",
+  "items": [
+    "Antonball Classic",
```

Launch script name with the extension :)
```suggestion
    "Antonball Classic.sh",
```